tchannel/testing/vcr/cassette.py

```diff
@@ -168,14 +177,19 @@ def _find(self, request, haystack=None):
                 yield interaction
 
     def _load(self):
+        file_hash = None
         try:
             with open(self.path, 'rb') as f:
                 data = f.read()
                 self.existed = True
+                file_hash = sha256(data).hexdigest()
+                if (self.path, file_hash) in self._cache:
+                    self._available = deepcopy(self._cache[(self.path, file_hash)])
```

Comment:
Nit: This can be simplified to,

```python
cached = self._cache.get((self.path, file_hash))
if cached is not None:
  self._available = deepcopy(cached)
  return
```